ssl/ssl_lib.c

```diff
@@ -2915,7 +3018,15 @@ SSL_CTX *SSL_CTX_new(const SSL_METHOD *meth)
     if (ret->ctlog_store == NULL)
         goto err;
 #endif
+
+    if (!SSL_CTX_set_ciphersuites(ret,
+            "TLS_AES_256_GCM_SHA384:"
+            "TLS_CHACHA20_POLY1305_SHA256:"
+            "TLS_AES_128_GCM_SHA256"))
+        goto err;
```

Comment:
That default string should be coming out of a define in ssl.h - i.e. TLS_DEFAULT_CIPHERSUITES to match SSL_DEFAULT_CIPHER_LIST ?

Reply:
Good idea. Done.